ui/src/components/ClientsTable/index.jsx

```diff
@@ -31,7 +34,7 @@ export default class ClientsTable extends Component {
   };
 
   state = {
-    sortBy: tableHeaders[0],
+    sortBy: 'client id',
     sortDirection: 'asc',
   };
 
```

Comment:
Navigating to http://localhost:5080/auth/clients shows:

![Screen Shot 2019-10-23 at 2 57 29 PM](https://user-images.githubusercontent.com/3766511/67425144-7d2cae80-f5a5-11e9-8972-e9f1aedf547e.png)

On the production site, there's an arrow beside the client id column name:
![Screen Shot 2019-10-23 at 2 57 39 PM](https://user-images.githubusercontent.com/3766511/67425143-7d2cae80-f5a5-11e9-836e-9073c20dbb49.png)